client/src/components/pages/Edit.js

```diff
@@ -27,13 +31,45 @@ import "firebase/auth";
 import { createJobObject, checkIfEmpty } from "../../effects/filter.effects";
 
 const mockTechStackData = {
-  languages: ["Java", "JavaScript", "C++", "C"],
-  frameworks: ["React", "Angular", "HTML", "CSS"],
-  workTools: ["Jira", "Asana", "Confluence", "Notion"],
+  languages: [
+    "C",
+    "C#",
+    "C++",
+    "CSS",
+    "HTML",
+    "Java",
+    "JavaScript",
+    "MATLAB",
+    "Python",
+    "R",
+    "SQL",
+    "TypeScript",
+  ],
+  frameworks: [
+    "AWS",
+    "Angular",
+    "Bootstrap",
+    "Docker",
+    "Google Cloud",
+    "Linux",
+    "MongoDB",
+    "Node",
+    "React",
+    "Ruby on Rails",
+    "Unix",
+  ],
+  workTools: ["Azure", "GitHub", "Jira", "Jupyter"],
   csConcepts: [
-    "Object Oriented Programming",
+    "Agile Development",
+    "Algorithms",
+    "Asynchronous Programming",
+    "Data Structures",
+    "Design Principles & Patterns",
```

Comment:
What kind of Design Principles? Graphic designers have their own design principles. Might be ideal to add in `Software` in front of this

Reply:
per conversation with @tommytwm  this has been resolved